akka-projection-r2dbc/src/it/scala/akka/projection/r2dbc/R2dbcProjectionSpec.scala

```diff
@@ -84,7 +84,8 @@ object R2dbcProjectionSpec {
   }
 
   object TestRepository {
-    val table = "projection_spec_model"
+    // FIXME H2 wants table names to be UC?
+    val table = "PROJECTION_SPEC_MODEL"
```

Comment:
Didn't investigate this enough, will revisit, but the error went away when I upper cased it.

Reply:
I figured this one out now after some head-scratching. By default H2 uppercases identifiers, _unless_ they are quoted. In the DDL for creating this test table below, we quote the name, so it is left alone, but then we do not quote the table name in the queries, so those are upper-cased.

I initially set out to disable the upper-casing in persistence-r2dbc since that magic seems weird, but maybe we can just remove the quotes in the table creation here :)

Reply:
Or quote it everywhere, if it is needed for one of the other dbs (but I don't think it is?)